tests/acceptance/test_nidaqmx_measurement.py

```diff
@@ -0,0 +1,112 @@
+import math
+import pathlib
+from typing import Generator, Iterable, NamedTuple
+
+import pytest
+
+from ni_measurementlink_service._internal.stubs.ni.measurementlink.measurement.v2.measurement_service_pb2 import (
+    MeasureRequest,
+)
+from ni_measurementlink_service._internal.stubs.ni.measurementlink.measurement.v2.measurement_service_pb2_grpc import (
+    MeasurementServiceStub,
+)
+from ni_measurementlink_service._internal.stubs.ni.measurementlink.pin_map_context_pb2 import (
+    PinMapContext,
+)
+from ni_measurementlink_service.measurement.service import MeasurementService
+from tests.assets.stubs.nidaqmx_measurement.types_pb2 import (
+    Configurations,
+    Outputs,
+)
+from tests.utilities import nidaqmx_measurement
+from tests.utilities.pin_map_client import PinMapClient
+
+_SITE = 0
+
+
+def test___single_session___measure___returns_measured_values(
+    pin_map_context: PinMapContext,
+    stub_v2: MeasurementServiceStub,
+) -> None:
+    pin_names = ["Pin1"]
+    configurations = Configurations(pin_names=pin_names, multi_session=False)
+
+    outputs = _measure(stub_v2, pin_map_context, configurations)
+
+    assert math.isclose(outputs.voltage_values[0], -0.14969328897976622, rel_tol=1e-9)
```

Comment:
```
================================== FAILURES ===================================
__________ test___single_session___measure___returns_measured_values __________

pin_map_context = pin_map_id: "C:\\actions-runner\\1\\_work\\measurementlink-python\\measurementlink-python\\tests\\assets\\acceptance\\session_management\\1Daqmx2ChannelGroup2Pin1Site.pinmap"
sites: 0

stub_v2 = <ni_measurementlink_service._internal.stubs.ni.measurementlink.measurement.v2.measurement_service_pb2_grpc.MeasurementServiceStub object at 0x06E78CD0>

    def test___single_session___measure___returns_measured_values(
        pin_map_context: PinMapContext,
        stub_v2: MeasurementServiceStub,
    ) -> None:
        pin_names = ["Pin1"]
        configurations = Configurations(pin_names=pin_names, multi_session=False)
    
        outputs = _measure(stub_v2, pin_map_context, configurations)
    
>       assert math.isclose(outputs.voltage_values[0], -0.14969328897976622, rel_tol=1e-9)
E       assert False
E        +  where False = <built-in function isclose>(0.019074068422498244, -0.14969328897976622, rel_tol=1e-09)
E        +    where <built-in function isclose> = math.isclose

tests\acceptance\test_nidaqmx_measurement.py:36: AssertionError
```

This is far too specific. Simulated DAQmx devices return a sine wave with random noise. Real X Series devices like the one in my test machine can return any value between -10.5 and +10.5.